apps/local_laplacian/local_laplacian_generator.cpp

```diff
@@ -165,6 +165,17 @@ class LocalLaplacian : public Halide::Generator<LocalLaplacian> {
             gray.add_trace_tag(cfg.to_trace_tag());
         }
 
+        {
```

Comment:
I use the local laplacian video in talks - I don't want to make it look weird. Maybe use the tone curve in camera pipe instead?